src/imageTools/mouseButtonRectangleTool.js

```diff
@@ -22,6 +22,9 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneMath, cornerstoneTo
         function addNewMeasurement(mouseEventData)
         {
             var measurementData = mouseToolInterface.createNewMeasurement(mouseEventData);
+            
+            //prevent adding new measurement if tool returns nill
+            if (!measurementData) return;
```

Comment:
Can you make this

````` javascript
if (measurementData === undefined) {
    return;
}
````

Since this may be used for tools other than Highlight later on, and it will keep things consistent.